apps/src/templates/ContentContainer.jsx

```diff
@@ -49,7 +52,10 @@ class ContentContainer extends Component {
     return (
       <div style={[boxStyles, bottomMargin]}>
         {(heading || (link && linkText)) && (
-          <div style={styles.headingBox}>
+          <div
+            className={classNames(moduleStyles.contentContainerHeading)}
```

Comment:
nit: don't need to use `classNames` if there's only one class

Reply:
Fixed. Thanks a lot for the hint!